float32[] sensorFT
uint16 sensorCounter
